datastore/redis.go

```diff
@@ -86,6 +83,7 @@ type RedisCache struct {
 	// prefixes (keys generated with a function)
 	prefixGetHeaderResponse           string
 	prefixExecPayloadCapella          string
+	prefixPayloadContentsDeneb        string
```

Comment:
it's okay with me, but why not follow the prev pattern (`prefixExecPayloadCapella` -> `prefixExecPayloadDeneb`)?

Reply:
I can answer this one. The type has changed to include the payload & blobs. `PayloadContents` is accurate.